src/index.js

```diff
@@ -0,0 +1,7 @@
+import Game from './game';
+
+window.addEventListener('load', () => {
+  const size = 4;
```

Comment:
let's name `default_size` , as we want to implement selector with other sizes in future